mcux/mcux-sdk/devices/MIMX9352/drivers/fsl_edma_soc_rev2.h

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Can you, please, add a similar file for 8QM?
You said you tested 8QM with EDMA and is ok, right?
IMO is better to added it now since 8QM is also a "user" of revision 2 of the edma driver.

Reply:
> You said you tested 8QM with EDMA and is ok, right?

Yes.


> IMO is better to added it now since 8QM is also a "user" of revision 2 of the edma driver.

The main issue I have with this is the fact that it will increase the time until https://github.com/zephyrproject-rtos/zephyr/pull/65671 can be merged, which, in turn, delays future work on the EDMA driver (for instance, I have some QM-related fixes for the Zephyr driver which I can't upstream at the moment since the driver hasn't been merged yet). If it's alright with you I'd like to get the driver out of the way as quickly as possible. This way I can get the fixes upstream and add support for new platforms in parallel.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/65671 this is already dependent on this PR which is not merged, yet.
For QM you'll also have to go through the same cycle - update hal, update Zephyr, etc. 
Maybe, this way we can skip the "update hal" part?
At least, this is what I thought.

Reply:
> For QM you'll also have to go through the same cycle - update hal, update Zephyr, etc.

Exactly, but the pipeline also implies updating MCUX-SDK so it would look more like this:

Update MCUX-SDK ---> Update HAL ---> Update Zephyr

This means having to involve another step in this, which is a source of delay. From my previous experience, the main sources of delay are the MCUX-SDK and Zephyr steps. If we were to also add the QM part into this patch we would skip the "Update HAL" step like you said but this step has never been a great source of delay in the pipeline anyways so we're fixing a problem which is not that impactful.

Reply:
Ok, I understand.
For 8QM you have more files to update/create so you still end up with mcux-sdk->hal->zephyr.